### ISSUE TITLE
When slides count less 4

### ISSUE THREAD

**Issue by EL-shadow** [NikolajDL/Excolo-Slider#12]
3 slide when scrolling is strange effect

**Comment by MemoryBug** [comment]
yes, I also found this issue, it annoy me, do you find the solution about it ?

**Comment by EL-shadow** [comment]
I tried fix issue in this commit https://github.com/EL-shadow/Excolo-Slider/commit/6114797cb63acf0165f495c8879f6bfce47e9ede
in my fork

**Comment by MemoryBug** [comment]
thank you for your reply

2015-01-30 20:23 GMT+08:00 EL-shadow <notifications@github.com>:
